src/openrct2-ui/CMakeLists.txt

```diff
@@ -75,6 +75,11 @@ if (WIN32)
     # tell it that it is
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D__USE_MINGW_ANSI_STDIO=1")
 endif ()
+
+if (PORTABLE)
+    set(CMAKE_INSTALL_RPATH "$ORIGIN")
```

Comment:
CMAKE uses `${...}` for variables and `$ENV{...}` for environment variables. I think this is saw on various internet pages for how to do it in cmake.